### PR TITLE
Poking around and messing around with the Ancient Grain artifacts

### PR DESCRIPTION
**Pull request by rhermida**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by maimorag** [comment]
> [CLA assistant check](https://cla-assistant.io/demisto/content?pullRequest=33673) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=33673) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=33673) it.

Hi @rhermida ,
Thank you for your contribution!
Please sign the CLA so I can start reviewing your contribution :)

File: Packs/AncientGrain/README.md
Comment (ssokolovich) [inline Packs/AncientGrain/README.md:1]: Please add some sentences about the pack. It is always helpful to describe the pack's intention.

File: Packs/AncientGrain/Playbooks/RH_-_ChatGPT_Ancient_Grain_Recipes.yml [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+        simple: Provide a recipe using the ancient grain of ${incident.marcancientgrain}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Perplexity LLM Custom
+      description: Send Text Message as a prompt to Perplexity
+      id: 38757eb4-8751-4b0b-8bd5-2d3c76396543
+      iscommand: true
+      name: Send Ancient Grain to ChatGPT for Recipe
+      script: Perplexity LLM Custom|||perplexity-send-prompt
```
Comment (ssokolovich) [inline Packs/AncientGrain/Playbooks/RH_-_ChatGPT_Ancient_Grain_Recipes.yml:206]: didn't find this command in our repository - can you please share more details on this?

**Review by ssokolovich — CHANGES_REQUESTED** [review]
Hey @rhermida, Thanks for your contribution to our Marketplace. 
Before we proceed, please sign your CLA so we can proceed forward with this Pull Request.

Adding below my notes on your work:
1. Assuming that the `TestMalware` pack is indeed a Test of yours. Please mention if that is intentional or by mistake. In case it was added by mistake, please remove it from this PR.
2. Added some comments on your contributed content- please review them. Feel free to mention me or reach out through our DFIR if you have any questions.

Moving forward (after the CLA), I would like to see a demo of this content.

**Comment by maimorag** [comment]
Hi @rhermida, we haven’t heard from you in a while.
Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

**Comment by maimorag** [comment]
Hi @rhermida, 
Thank you for your contribution. 
I have not received a response from you in the last two weeks, so I will close your PR now.
Please feel free to re-open it when you are available to continue. 
Thanks again.

**Pull request closed without merging**